src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -17,6 +17,9 @@ import {
   EventReporter,
   VideoDownlinkBandwidthPolicy,
   Logger,
+  Device,
+  AudioTransformDevice,
+  isAudioTransformDevice
```

Comment:
Nit: trailing commas on all of your import lists.

Reply:
Updated